apps/verify/views/content/not-found-brp.md

```diff
@@ -0,0 +1,3 @@
+The biometric residence permit (BRP) number and date of birth must match what is on our system.
+
+Check your BRP and date of birth, then try again. Or try signing in with your unique application number.
```

Comment:
Remove "Or try signing in with your unique application number." it has changed on figma